### PR TITLE
fix some bugs in docs and translate some files from en to zh

### PR DESCRIPTION
**Pull request by qzfzz**
Hi!
I have done some translation from en to zh and fix some bugs.Pease check and pull the source code.

### CODE REVIEW

File: en/exceptions.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ When an exception is thrown, a "catch" block can be used to capture the exceptio
 developer to provide proper handling.
 
 .. code-block:: zephir
-
+	var e;
```
Comment (sergeyklay) [inline en/exceptions.rst]: can you fix indents related issues. here and below

**Comment by sergeyklay** [comment]
@dreamsxin Could you please take a look

File: en/exceptions.rst (hunk 1 above)
Comment (qzfzz) [inline en/exceptions.rst]: I'm so sorry for making you so much trouble, I have fixed it now.

**Comment by qzfzz** [comment]
I'm so sorry for making you so much trouble, I have already fixed the issues.

**Comment by qzfzz** [comment]
Hi, I have fixed the indent issues, and do some translation work from en to zh! Please check and do pull from repo.

File: en/warnings.rst [hunk 2]
```diff
@@ -156,16 +156,3 @@ Raised when the compiler detects that an element is being tried to be appended t
...
-Raised when the compiler detects that an element is being tried to be appended to a non array variable:
-
-.. code-block:: zephir
-
-    public function some()
-    {
-        var b = false;
-        let b[] = "some value";
-    }
-
```
Comment (sergeyklay) [inline en/warnings.rst]: Could you please explain?

Comment (qzfzz) [inline en/warnings.rst]: duplicated paragraph

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**